.github/workflows/tests.yml

```
@@ -0,0 +1,91 @@
+      - name: Download Playwright test results
+        uses: actions/download-artifact@v4
+        with:
+          name: test-results
+          path: ./test-results
+      - name: Run Chromatic
+        uses: chromaui/action@latest
+        with:
+          playwright: true
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+        env:
+          CHROMATIC_ARCHIVE_LOCATION: playwright-report/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)